.mypy.ini

```diff
@@ -20,6 +20,7 @@ warn_unused_ignores = True
 disallow_any_unimported = True
 warn_return_any = True
 exclude = .*_pb2.py
+ignore_missing_imports = True
```

Comment:
why is this ?

Reply:
error: Skipping analyzing "semantic_version": module is installed, but missing library stubs or py.typed marker  [import-untyped]

https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports